server/src/models/equipment/reservationInput.ts

```diff
@@ -1,8 +1,10 @@
 export interface ReservationInput {
-    userId: number;
-    supervisorId: number;
-    machineId: number;
-    createdAt: Date;
+
+    creator: number;
```

Comment:
I don't think we need `creator` since we can get that from the context.

Reply:
@MatthewGalan you added `creator` and `creatorID` in the types. https://github.com/rit-construct-makerspace/makerspace/pull/125/files I was following this. Should we get rid of this?

Reply:
Good catch. That PR was before I knew about how the context works. Yeah I still think we can remove it.